tests.py

```diff
@@ -1766,6 +1796,21 @@ def test_suffix_with_periods_with_lastname_comma(self):
         self.m(hn.last, "Doe", hn)
         self.m(hn.suffix, "Msc.Ed.", hn)
 
+    @unittest.SkipTest
+    def test_suffix_in_nickname_dup(self):
```

Comment:
I'll look at this.  I might have added these tests for my initial approach to the problem.  Once I changed my approach, I just skipped the test rather than delete the (test_) function.  Looking at the code will jog my memory.

I'm going to ask a retired judge about the titles that one might apply to his profession.  I'll email you after we've talked.